plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/model/EntityDiagram.java

```diff
@@ -200,4 +205,13 @@ public boolean isDirty() {
     public void setDirty(boolean dirty) {
         this.dirty = dirty;
     }
+
+    public ERDNotationDescriptor getDiagramNotation() {
+        return notationStyle;
+    }
+    
+    public void setDiagramNotation(ERDNotationDescriptor notation) {
+        this.notationStyle = notation;
+    }
```

Comment:
Could you cover methods with annotations, it will help us later to understand the nullability of values

Reply:
I don't see any changes.